Resizing works when selection box drawn in any way

derekgygax opened the pull request:
Resizing the selection box will work when drawn in any way.
Top, left to down, right
top, right to down, left
bottom right to up, left
bottom left to up, right

derekgygax commented:
I have not used npm gulp watch to build a distribution version yet.

MindFreeze commented:
I haven't worked on this in years, so I am not sure about this, but shouldn't the fix be in `onOutsideDrag` instead of `onOutsideDragEnd` since the original resizing code is in `onOutsideDrag` ?
Also you should build a distribution version for the PR to be accepted.

derekgygax commented:
I think having the changes in ```onOutsideDragEnd``` and the new function I just added ``` onBorderDragEnd ``` does everything you need. Originally resizing didn't work because the point you start drawing at was always treated as the top left and the deltas were calculated from that. The functions I changed/added recalculate what is considered the top and left after you finish drawing or resizing. What do you think? Does that sound right?

MindFreeze commented:
Ah, now I see what you are doing. This should be ok.

dthornley commented:
Let me know when you've added the updated distribution package and I'll merge this.
